src/test/java/com/rtb/projectmanagementtool/user/UserControllerTest.java

```diff
@@ -12,7 +12,41 @@
 import org.junit.Test;
 
 public class UserControllerTest {
+
+  // User 1
+  private static final long userID1 = 1l;
+  private static final long AuthID1 = 1l;
+  private static final String name1 = "Anna";
+  private static final long year1 = 2023;
+  private static final ArrayList<String> majors1 = new ArrayList<>(Arrays.asList("Biology", "Gov"));
+  private static final long total1 = 3;
```

Comment:
nit: total what's this a total of?

Reply:
total completed tasks. I'll change it in my next PR.